api/tests/e2e/service_instances_test.go

```diff
@@ -123,13 +123,15 @@ var _ = Describe("Service Instances", func() {
 			})
 
 			It("deletes the service instance", func() {
```

Comment:
This appears unrelated to the story. Maybe better in a separate PR?

Reply:
This was only because i couldn't get it to run locally. It doesn't seem to be an issue when git runs the e2e test so I reverted it.